packages/paste-core/components/date-picker/stories/index.stories.tsx

```diff
@@ -140,11 +188,13 @@ export const InverseReadonlyDatePicker: React.FC = (props) => {
   return (
     <Box backgroundColor="colorBackgroundBodyInverse" padding="space60">
       <Label htmlFor={uidDP} variant="inverse">
-        View account insights from the following date:
+        In what month was the Combahee River Collective Statement published?
```

Comment:
NIT: Should this be "Combahee River Collective (CRC)" so the acronym in the anchor text is more clear?

Reply:
Made these changes, thanks for catching!